core/domain/story_domain.py

```diff
@@ -35,6 +37,7 @@
 STORY_PROPERTY_TITLE = 'title'
 STORY_PROPERTY_THUMBNAIL_BG_COLOR = 'thumbnail_bg_color'
 STORY_PROPERTY_THUMBNAIL_FILENAME = 'thumbnail_filename'
+STORY_PROPERTY_THUMBNAIL_SIZE_IN_BYTES = 'thumbnail_size_in_bytes'
```

Comment:
Per Sean's inputs, we don't need this property anymore.
The update happens together for thumbnail_filename and thumbnail_size_in_bytes.

Reply:
updated